### DISCUSSION TITLE
Negotiated Rates Reporting Period

### DISCUSSION THREAD

**Discussion by Kvillafor** [CMSgov/price-transparency-guide#418] [Q&A] [answered]
Is there a look up period for reporting on the negotiated rates file?  The Allowed Amounts file required us to report allowed and paid amounts during the 90-day period that begins 180 days prior to the publication date of the file.  For the negotiated rates file, do we report all negotiated rates for each procedure code and provider where services were rendered, within a specific time frame?  @shaselton-usds

**Comment by rakeshkatte** [comment, marked as answer]
Our understanding is, we need to report pricing for all codes effective as
of the day we generate the file and does not depend on claims received.

**Reply by Kvillafor** [reply]
Is your understanding that we have to report pricing for all codes for all/each contracted provider with the carrier? @rakeshkatte

**Comment by shaselton-usds** [comment]
@Kvillafor The Allowed Amounts and In-Network files are fundamentally two different things. One file is a retrospective look back (Allowed Amounts) and the other file is a prospective look forward (In-Network).

@rakeshkatte is correct in that the In-Network file isn't dependent on claims received (this is the Allowed Amounts file) but rather the contractual rates that are in place for all items/services that the plan offers.

Also @rakeshkatte mentioned that those contractual rates need to be accurate at the time of generating the In-Network file; this is also correct.

**Reply by Kvillafor** [reply]
When you say the contractual rates for all items/services that the plan offers, is that for every provider the carrier network contracts with?  As a third-party administrator, we are being asked to create the INN file for the carrier, but we would be basing this off the claims file we receive from the carrier.  We would only be able to report for any items or services for the rendering provider that came over on the claims file.  I don't believe that this would provide a complete picture, since we would not be capturing all negotiated rates for each item or service offered by the carrier network and each provider they contract with and thus, the file may not be compliant. Thoughts? @shaselton-usds
